.github/workflows/cypress-testing.yml

```
@@ -115,29 +133,196 @@ jobs:
+          SERVER_GRPC_INSECURE: 't'
+          SERVER_GRPC_BROADCAST_ADDRESS: 127.0.0.1:7077
+          SERVER_GRPC_PORT: '7077'
+          SERVER_URL: http://127.0.0.1:8888
+          SERVER_AUTH_SET_EMAIL_VERIFIED: 't'
+          SERVER_DEFAULT_ENGINE_VERSION: 'V1'
+
+    steps:
+      - run: echo "Running Cypress tests with Cloud recording for non-draft PR or push to v3"
+
+      - name: Define node version
+        uses: actions/setup-node@v3
```

actionlint (1.7.7)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)